graphql/schema.graphql

```diff
@@ -23,8 +23,17 @@ type Query {
   # User
   getUser(id: ID!): User @cacheControl(scope: PRIVATE)
   getUserByEmail(email: String!): User @cacheControl(scope: PRIVATE)
-  listUsers(organizationId: ID, costCenterId: ID, roleId: ID): [User]
-    @cacheControl(scope: PRIVATE, maxAge: SHORT)
+
+  listUsers(
```

Comment:
To avoid a breaking change (since some B2B clients may be using this query) I would suggest that we keep `listUsers` with the original functionality, and add a new `listUsersPaginated` query. We can add the `@deprecated` directive to `listUsers`, telling users to migrate to the new query. Something like `@deprecated(reason: "This query is deprecated, use listUsersPaginated query instead.")`

Reply:
Awesome! I'm gonna work in this change, it really makes sense